### PR TITLE
vmui: fix handling invalid timezone

### PR DESCRIPTION
**Pull request by Loori-R**
- Add handling for an invalid browser timezone. If it's impossible to determine the timezone, set it to `UTC`.
- Add a warning to the timezone selection if the browser timezone couldn't be determined.

#5732 

image

### CODE REVIEW

File: app/vmui/packages/vmui/src/utils/time.ts [hunk 1]
```diff
@@ -227,3 +227,22 @@ export const getTimezoneList = (search = "") => {
...
+  } catch (e) {
+    return false;
+  }
+};
+
+export const getBrowserTimezone = () => {
+  const timezone = dayjs.tz.guess();
+  const isValid = isValidTimezone(timezone);
+  return  {
+    isInvalid: !isValid,
```
Comment (dmitryk-dk) [inline app/vmui/packages/vmui/src/utils/time.ts:244]: I think it is better to make the field consistent and do not do reverse logic. I suggest to use `IsValid` field in the object and use it in other places

**Review by dmitryk-dk — COMMENTED** [review]
See my comment

File: app/vmui/packages/vmui/src/utils/time.ts (hunk 1 above)
Comment (Loori-R) [reply app/vmui/packages/vmui/src/utils/time.ts]: updated

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
@Loori-R , thanks for the fix!

**Pull request merged**